OAuth/FirebaseOAuthUI/FUIOAuth.m

```diff
@@ -191,11 +191,15 @@ + (FUIOAuth *)yahooAuthProvider {
 }
 
 + (FUIOAuth *)appleAuthProvider {
+  return [self appleAuthProviderWithUserInterfaceStyle:UITraitCollection.currentTraitCollection.userInterfaceStyle];
```

Comment:
Does this need an availability check?

Reply:
Hi, @morganchen12.   
Because we are using [`ASAuthorizationAppleIDRequest`](https://developer.apple.com/documentation/authenticationservices/asauthorizationappleidrequest?language=objc) internally, it needs availability check.

https://github.com/firebase/FirebaseUI-iOS/blob/5e7e9bcb6b6674effdd5e8cc0de4b138bdfd5d7c/OAuth/FirebaseOAuthUI/FUIOAuth.m#L255-L263